homeassistant/components/ai_speaker/sensor.py

```diff
@@ -65,8 +66,16 @@ def unit_of_measurement(self) -> str:
 
     @property
     def device_state_attributes(self):
-        """Return the attributes of the sensor."""
-        return self._ais_info
+        """Return the attributes of the device."""
+        return {
+            "ais_id": self._ais_id,
+            "ais_product": self._ais_product,
```

Comment:
We only want dynamic info related to the state of the entity in the state attributes. Only exception can be static info that is essential for automations. Looks like most of this info is static and not useful for automations.

Maybe the id is useful? Please review each attribute and remove the ones that don't fit our design criteria.

Reply:
OK, the static attributes are removed, 
Only 3 attributes left - that can change and can be useful for automation.

Reply:
That information can't really be useful ? Nothing here seems like it's user facing at all ?

Reply:
Hi Master :) Of course you are right api level it is not so useful for the AI-Speaker end user.

The first idea was to have all the info about AI-Speaker device in this sensor as a suplement to device info in HA. But then (thanks to @MartinHjelmare ) I learned that HA in sensor want to have only "dynamic info related to the state of the entity"  usfull in automation.

In fact we can give something more usfull then api level for end user - I'm going to add as the sensor value the speaker Internet connection spead. This information should be helpfull for user. The api level will go to attributes - hope this will be OK for you.

The real functionality of this integration will be delivered later (in next pull requests) course as I learned the big pull requests are not acceptable from beginners.

Thanks guys for you help with my first HA pull. :+1: 
Let me say that HA project and your approach is awesome. This is the best code I saw in my entire programing career :1st_place_medal:  big respect! 
I'm learning fast and hope to be some day the member of the HA core team :)